packages/@stylexjs/postcss-plugin/package.json

```diff
@@ -8,12 +8,28 @@
     "url": "git+https://github.com/facebook/stylex.git"
   },
   "license": "MIT",
+  "scripts": {
+    "test": "jest"
+  },
+  "jest": {
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/__fixtures__/"
+    ],
+    "testEnvironment": "node"
+  },
```

Comment:
Please move this into a jest config file (like the other packages)

Reply:
Fixed in https://github.com/facebook/stylex/pull/1067/commits/a73a2c9dd0d87b44a7a8a38f75d2cef2d237063b